Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py

```diff
@@ -20,3 +20,32 @@
                                             + SiPixelPhase1TrackingParticleAnalyzerV
                                             )
 
+###PixelTracking only configurations for gpu wf###
+#digi
+pixelOnlyDigisAnalyzerV = SiPixelPhase1DigisAnalyzerV.clone()
+#cluster
+pixelOnlyTrackClustersAnalyzerV = SiPixelPhase1TrackClustersAnalyzerV.clone()
+pixelOnlyTrackClustersAnalyzerV.clusters = cms.InputTag('siPixelClustersPreSplitting')
+pixelOnlyTrackClustersAnalyzerV.tracks = cms.InputTag('pixelTracks')
```

Comment:
```suggestion
pixelOnlyTrackClustersAnalyzerV = SiPixelPhase1TrackClustersAnalyzerV.clone(
    clusters = 'siPixelClustersPreSplitting',
    tracks = 'pixelTracks'
)
```

Reply:
I have been told to prefer this syntax when making changes to a module after cloning it.